apps/activitypub/src/views/Explore/Explore.tsx

```
@@ -1,17 +1,206 @@
+                        <div className='inline-flex items-center'>
+                            <Skeleton className='w-24' />
+                        </div>
+                    )}
+                </div>
+                <div className='pl-[52px]'>
+                    {isLoading ?
+                        <Skeleton className='w-full max-w-96' />
+                        :
+                        profile.bio &&
+                        <div
+                            dangerouslySetInnerHTML={{__html: openLinksInNewTab(stripHtml(profile.bio, ['a', 'br']))}}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)